lib/reply.js

```diff
@@ -115,10 +122,21 @@ Reply.prototype.send = function (payload) {
   }
 
   if (payload instanceof Error || this[kReplyIsError] === true) {
+    // If the reply was already hijacked, we skip sending the error since
+    // the caller is expected to handle it depending on their use-case
+    if (this[kReplyHijacked] === true) {
+      return this
+    }
+
     onErrorHook(this, payload, onSendHook)
     return this
   }
 
+  // If the reply was hijacked, we skip sending
+  if (this[kReplyHijacked] === true) {
+    return this
+  }
```

Comment:
could you just move this if before the error block?

Reply:
True! I had made some changes but then forgot to move it back after I reached the current version.